### ISSUE TITLE
allDay support for events

### ISSUE THREAD

**Issue by jasonmorehouse** [Appgutta/AGCalendar#36]
When adding an event with no time, 00:00 -- 06:00 is listed for the time for the event when clicking on a date.  Is there anyway to hide the time incase of a fullday event?

I noticed in the code it checks for allDay, but passing this to the event doesn't seem to make a difference.

Eg:

```
    Ti.Calendar.addEvent({
                title: "My all day event",
                startDate: date,
                endDateDate: date,
                allDay: true,
   });
```

![Screen Shot 2013-03-11 at 12 51 25 PM](https://f.cloud.github.com/assets/1195376/245545/145ddbd2-8a85-11e2-96a6-1feded36836d.png)

Thanks for all of the improvements.  Very handy library.

**Comment by ChrisRM** [comment]
allDay event are currently not supported, but I've added it in version 2.0 ( See https://github.com/Appgutta/AGCalendar/issues/35#issuecomment-14678799 ).

**Comment by jasonmorehouse** [comment]
Sounds great.  Let me know when 2.0 is ready for some testing.  Thanks for the efforts.

**Comment by ChrisRM** [comment]
Fixed in version 2.0, will be merged in version 1.3
